### DISCUSSION TITLE
polyfill.min.js能否配置使用cdn或本地，从官网加载将近2s

### DISCUSSION THREAD

**Discussion by steinvenic** [AlistGo/alist#2307] [General]
![image](https://user-images.githubusercontent.com/18524326/201461674-fced52dc-20bf-46d1-a095-426b22201ec6.png)

**Comment by anwen-anyi** [comment]
https://polyfill.alicdn.com/v3/polyfill.min.js?features=String.prototype.replaceAll

你试试这个？阿里的好像
